src/components/NewsRoomList.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import FilterList from "./FilterList";
+import NewsRoomCard from "./NewsRoomCard";
+import { newsItems } from "../files/NewsItems";
+import ListCounter from "./ListCounter";
+import useNewsRoom from "../hooks/useNewsRoom";
+import { Alert, Button } from "@baltimorecounty/dotgov-components";
+
+const NewsRoomList = () => {
+  const { hasError, newsRoomItems = [], isLoading } = useNewsRoom();
+  const [filteredItems, setFilteredItems] = useState([]);
+
+  const handleLoadMoreNewsItems = clickEvent => {
+    setFilteredItems([]);
+    console.log("Load More Results");
+  };
+
+  if (hasError) {
+    return (
+      <Alert className="status" type="error">
+        <p>
+          Unable to retrieve the list of Baltimore County news room. Please try
+          again in a couple of minutes.
+        </p>
+      </Alert>
+    );
+  }
+
+  return (
+    <React.Fragment>
+      {isLoading ? (
+        <p>Loading Baltimore County News...</p>
+      ) : (
+        <div className="row">
+          <div className="col-md-6 col-xs-12 order-xs-last order-md-first ">
+            <ListCounter
+              count={
+                filteredItems.length === 0
+                  ? newsRoomItems.length
+                  : filteredItems.length
+              }
+              total={newsRoomItems.length}
+            />
+          </div>
+          <FilterList
+            items={newsItems}
+            renderItem={props => (
+              <div
+                className="d-flex col-lg-12 col-md-12 col-sm-12"
```

Comment:
if the card is full width, i'm not sure we need the bootstrap classes here.

If we do, an way to make this cleaner is `className="d-flex col-xs-12"` this will apply 12 columns to all screen sizes. I think `className="d-flex col-12"` might even work, but i am not as familiar with this latest bootstrap grid as the previous non flexbox version.

Reply:
changed to col-12

Reply:
@tmccoy529 I'm not seeing this one?

Reply:
must have gotten reverted. When I do my next push it should be there.